packages/launchpad-ledger/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@cosmjs/launchpad-ledger",
+  "version": "0.22.2",
+  "description": "A library for interacting with the Cosmos Launchpad Ledger Nano App",
+  "contributors": [
+    "Will Clark <willclarktech@users.noreply.github.com>"
+  ],
+  "license": "Apache-2.0",
+  "main": "build/index.js",
+  "types": "types/index.d.ts",
+  "files": [
+    "build/",
+    "types/",
+    "*.md",
+    "!*.spec.*",
+    "!**/testdata/"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/CosmWasm/cosmjs/tree/master/packages/launchpad-ledger"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "docs": "typedoc --options typedoc.js",
+    "format": "prettier --write --loglevel warn \"./src/**/*.ts\"",
+    "format-text": "prettier --write --prose-wrap always --print-width 80 \"./*.md\"",
+    "lint": "eslint --max-warnings 0 \"**/*.{js,ts}\"",
+    "lint-fix": "eslint --max-warnings 0 \"**/*.{js,ts}\" --fix",
+    "move-types": "shx rm -rf ./types/* && shx mv build/types/* ./types && rm -rf ./types/testdata && shx rm -f ./types/*.spec.d.ts",
+    "format-types": "prettier --write --loglevel warn \"./types/**/*.d.ts\"",
+    "prebuild": "shx rm -rf ./build",
+    "build": "tsc",
+    "postbuild": "yarn move-types && yarn format-types",
+    "build-or-skip": "[ -n \"$SKIP_BUILD\" ] || yarn build",
+    "test-node": "node jasmine-testrunner.js",
+    "test-firefox": "yarn pack-web && karma start --single-run --browsers Firefox",
+    "test-chrome": "yarn pack-web && karma start --single-run --browsers ChromeHeadless",
+    "test": "yarn build-or-skip && yarn test-node",
+    "coverage": "nyc --reporter=text --reporter=lcov yarn test --quiet",
+    "pack-web": "yarn build-or-skip && webpack --mode development --config webpack.web.config.js"
+  },
+  "dependencies": {
+    "@cosmjs/utils": "^0.22.2",
+    "@ledgerhq/hw-transport-webhid": "^5.23.0",
```

Comment:
So far! It's used by Lunie for Windows support. Do we need that?

Reply:
Windows support is blocked by

```ts
  if (isWindows(platform)) {
    throw new Error("Windows is not currently supported.");
  }
```

I'd do

1. Support WebUSB only
2. Figure out if we can use WebUSB with Windows
3. Add Node.js support as a 2nd transport
4. Add additional transports on demand

Reply:
Lunie has code for Windows support but they use webhid. I just put that error in to simplify things and because I have no way to test it.

https://github.com/luniehq/cosmos-ledger/blob/develop/src/cosmos-ledger.ts#L85-L96

Reply:
I see, thanks. However, it is well possible that the lack of Windows support for WebUSB is outdated.

Reply:
https://github.com/LedgerHQ/ledgerjs/tree/master/packages/hw-transport-webusb#support-status

> Support status
WebUSB is currently only supported on Google Chrome / Chromium.
In Linux, user need to install the specific udev rules
In Mac, it should work.
In Windows, WebUSB does not work out of the box but you can fix it with Zadig.
In Android Chrome it works.